src/Bridge/Twig/Extension/EnumExtension.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Greg0ire\Enum\Bridge\Twig\Extension;
+
+use Greg0ire\Enum\AbstractEnum;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class EnumExtension extends \Twig_Extension
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(TranslatorInterface $translator = null)
+    {
+        $this->translator = $translator;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters()
+    {
+        return [
+            new \Twig_SimpleFilter('enum_label', [$this, 'label']),
+        ];
+    }
+
+    /**
+     * Displays the label corresponding to a specific value of an enumeration.
+     *
+     * @param mixed       $value             Must exist in the enumeration class specified with $class
+     * @param string      $class             The enum class name
+     * @param string|bool $translationDomain The translation domain to use if the translator if available.
+     *                                       string: Use the specified one
+     *                                       null: Use the default one
+     *                                       false: Do not use the translator
+     * @param bool|string $separator         Namespace separator to use with the class prefix.
+     *                                       - true: I want a separator, which implies to prefix with a class.
+     *                                       - 'ø': I want 'ø' as a separator, which implies to prefix with a class.
+     *                                       - false: I don't want any separator, because I don't even need a prefix.
+     *                                       By default, the value is the configured one if the translator
+     *                                       is available and enabled, false otherwise.
```

Comment:
After more consideration, this multi value argument does not look like such a good idea, sorry. Please split it back into two arguments ($classPrefix, boolean, and $separator).